osi_lane.proto

```diff
@@ -809,6 +808,36 @@ message LaneBoundary
         // See \c LaneBoundary .
         //
         optional double height = 3;
+
+        // Alternation of dashes in case of a dashed lane boundary. In
+        // context, this field gives information about the location of
+        // dashes on the boundary line.
+        //
+        optional Dash dash = 4;
+
+        // This enum describes the alternation of dashes in case of a
+        // dashed lane boundary.
+        //
+        // \note The enum descriptions adhere to the definition direction
+        // of the lane boundary points, meaning that start or end of a dash
+        // are understood with respect to the direction in which the points
+        // of the boundary line are defined.
+        //
+        enum Dash
```

Comment:
**Output CCB 18.08.2021:**
Please review the _enum_ guidelines for OSI, (Dash_UNKNOWN, etc..).

Reply:
Has been done. 
cc @HendrikAmelunxen